src/store.js

```diff
@@ -11,28 +10,13 @@ window.docomo || (window.docomo = {});
 /** eslint-enable */
 
 /** disable sentry in local env or by conf */
-if (process.env.LOCAL_DEV === true || !window.docomo.SENTRY_ENABLE) {
-  window.docomo.SENTRY_URL = null;
-}
-
-const SENTRY_URL = window.docomo.SENTRY_URL;
-const WHITE_LABEL = window.docomo.WHITE_LABEL;
-const B_TEST_ID = window.docomo.B_TEST_ID;
-const ENVIRONMENT = window.docomo.ENVIRONMENT;
-
-Raven.config(SENTRY_URL, {
-  release: version.build,
-  environment: ENVIRONMENT,
-  collectWindowErrors: true,
-}).install();
+// if (process.env.LOCAL_DEV === true || !window.docomo.SENTRY_ENABLE) {
+//   window.docomo.SENTRY_URL = null;
+// }
 
-Raven.setTagsContext({
-  label: WHITE_LABEL,
-  settrack: B_TEST_ID,
-});
 /* eslint-disable no-underscore-dangle */
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-const middlewares = [thunkMiddleware, trackingMiddleware];
+const middlewares = [createRavenMiddleware(Raven), thunkMiddleware, trackingMiddleware];
```

Comment:
You can add
`createRavenMiddleware(Raven, {})`
in line 19 (definition of _middlewares_ array) or line 29 (push on _middlewares_).
Remove one of these.